c2rust-transpile/src/translator/mod.rs

```diff
@@ -1644,10 +1649,27 @@ impl<'c> Translation<'c> {
                             _ => unreachable!("Found non-field in record field list"),
                         });
                 if has_bitfields {
-                    derives.push("BitfieldStruct");
                     self.use_crate(ExternCrate::C2RustBitfields);
                 }
 
+                let derives = self
+                    .tcfg
+                    .derives
+                    .iter()
+                    .flat_map(|derive| {
+                        let can_derive = match derive {
+                            Derive::Clone | Derive::Copy => !contains_va_list,
+                            Derive::Debug => can_derive_debug,
+                            Derive::BitfieldStruct => has_bitfields,
+                        };
+                        if can_derive {
+                            Some(derive.to_string())
+                        } else {
+                            None
+                        }
+                    })
+                    .collect_vec();
```

Comment:
```suggestion
                    .collect::<Vec<_>>();
```

Equally simple and doesn't require `itertools`; same with the other `.collect_vec()`.